docs/markdown/FAQ.md

```diff
@@ -405,6 +405,9 @@ advantages:
    not care what the extension is](https://docs.microsoft.com/en-us/cpp/build/reference/link-input-files?view=vs-2019),
    so specifying `libfoo.a` instead of `foo.lib` does not change the workflow,
    and is an improvement since it's less ambiguous.
+1. Autotools projects built with MinGW using UCRT are compatible with MSVC and
+   also name their static libraries `libfoo.a`. This is the naming convention
+   used by MSYS2, their UCRT libraries are usable with MSVC.
```

Comment:
The second sentence seems redundant, maybe change "projects built with MinGW" to "projects built with MinGW compilers such as MSYS2"?

Reply:
Good point, rephrased that a bit.

Reply:
> I might also raise it up to bullet point number 3

I think that point is internal to Meson, we use the same name on all platforms, which is nice but not really that important. I think I prefer keeping it separate.